src/test/java/com/puppycrawl/tools/checkstyle/bdd/InputConfiguration.java

```diff
@@ -75,7 +75,21 @@ public String getCheckName() {
     }
 
     public DefaultConfiguration createConfiguration() {
-        return new DefaultConfiguration(checkName);
+        final DefaultConfiguration parsedConfig = new DefaultConfiguration(checkName);
+        nonDefaultProperties.forEach(parsedConfig::addAttribute);
+        return parsedConfig;
+    }
+
+    public boolean isDefaultAttribute(String attribute) {
+        return defaultProperties.containsKey(attribute);
+    }
+
+    public String getDefaultAttributeValue(String attribute) {
```

Comment:
It is a little bit confusing when two `property` and `attribute` words are used
I suggest to select and use only one name, lets be consistent :)

Reply:
it seems `attribute` name makes more sense, what do you think?

Reply:
I think it as an ```attribute``` is a ```key-value``` pair and the key is called ```property```.

Reply:
generally, property is also key-value pair ex `System.getProperty`

Reply:
> it seems `attribute` name makes more sense, what do you think?

changed everywhere